src/components/modal/product-modal.tsx

```diff
@@ -0,0 +1,72 @@
+// import React from "react";
+import { useDispatch } from "react-redux";
+import { productModalDisplay } from "../../slices/modal";
+import './modal.scss';
+import ImageCard from "../image-card/image-card";
+
+const ProductModal = () => {
+    const dispatch = useDispatch();
+    const closeModal = (e:any) => {
+        e.stopPropagation();
+        dispatch(productModalDisplay())
+    }
+    // const arr = [1,2,3,4,5,6,7]
+    // const doNext = (index: number) => {
+    //     if(index > 0 && index < arr.length-1){
+    //         index++;
+    //         console.log(arr[index])
+    //     }
+    // }
+    // const doPrevious = (index: number) => {
+    //     if(index > 0 && index <= arr.length-1){
+    //         index--;
+    //         console.log(arr[index])
+    //     }
+    // }
+    // const keyEventHandler = (e: React.KeyboardEvent<HTMLDivElement>) => { 
+    //     if (e.code === 'Escape'){
+    //         closeModal(e);
+    //     }
+    // }
+    // const keyHandler = (e: React.KeyboardEvent<HTMLDivElement>) => {
+    //     if (e.code === 'tab'){
```

Comment:
Do we still need the commented out code? @onyi2020

Reply:
I am not sure, I think for the next task,otherwise, I will just take it out if it is not necessary